ingress-nginx.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Ingress NGINX Controller
+category: server-app
+tags: go kubernetes
+permalink: /ingress-nginx
+releasePolicyLink: https://github.com/kubernetes/ingress-nginx/security#supported-versions
+releaseDateColumn: true
+eolColumn: false
```

Comment:
the page isn't really [helpful without this](https://github.com/endoflife-date/endoflife.date/wiki/Guiding-Principles#be-helpful).

Reply:
all done